### PR TITLE
OCPQE-25059: Remove Hypershift Azure jobs with OCP MC

### PR DESCRIPTION
**Pull request by fxierh**
Since 4.17, hosting an Azure guest cluster from an OCP management cluster is no longer possible. Update our jobs accordingly. 

https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55816/rehearse-55816-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-azure-ipi-ovn-hypershift-guest-f14/1826207870534815744

### CODE REVIEW

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-azure-ipi-ovn-hypershift-guest-f7

**Comment by fxierh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-azure-ipi-ovn-hypershift-guest-f14

**Comment by fxierh** [comment]
/cc @liangxia 

Could you please take a look ? Thank you

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -351,6 +351,17 @@ workflows:
...
   cucushift-installer-rehearse-azure-ipi:
     platform: azure
+  cucushift-installer-rehearse-azure-ipi-ovn-hypershift-guest:
+    platform: azure-qe
+    base_images:
+      hypershift-operator:
+        name: "4.16"
+        namespace: ocp
+        tag: hypershift-operator
+      upi-installer:
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:361]: Does `azure-ipi-ovn-hypershift-guest` require `upi-installer` ?

Comment (fxierh) [reply core-services/ci-chat-bot/workflows-config.yaml:361]: Yep. `cucushift-hypershift-extended-enable-qe-catalogsource` and `cucushift-hypershift-extended-enable-qe-pull-secret` require upi-installer.

Comment (liangxia) [reply core-services/ci-chat-bot/workflows-config.yaml:361]: Ack

**Pull request merged**